CryptoPrices/Sources/Data/Sources/NetworkCore/CoinAPIProtocol.swift

```diff
@@ -11,6 +11,6 @@ public protocol CoinAPIProtocol {
     // TODO: Update when implement
     func myCoins() async throws -> [APICoin]
     func trendingCoins(coinIDs: [String]) async throws -> [APICoin]
-    func coinDetail() async throws -> APICoin
     func getChartPrices(coinID: String, numberOfDays: String) async throws -> APIPrices
+    func coinDetail(id: String) async throws -> APICoinDetail
```

Comment:
This is the last _Backend_ task (at least for the initial plan), I think it's time to remove the mark `// TODO: Update when implement` 🙏